docroot/modules/custom/uiowa_core/src/HeadlineHelper.php

```diff
@@ -40,8 +40,18 @@ public static function getHeadingSizeUp($size) {
   public static function getStyles() {
     return [
       'default' => 'headline block__headline',
-      'headline_bold_serif' => 'headline bold-headline bold-headline--serif block__headline',
-      'headline_bold_serif_underline' => 'headline bold-headline bold-headline--serif bold-headline--underline block__headline',
+      'headline_bold_serif' => 'headline headline--serif block__headline',
+      'headline_bold_serif_underline' => 'headline headline--serif headline--underline block__headline',
```

Comment:
This isn't strictly related to this functionality, but if there are any classes that we add every time (like `headline` and `block__headline`, we should maybe just add those when the headline is rendered rather than having them as part of these options.

Reply:
Added a todo for another day. Does that work for you @pyrello ?